.github/workflows/ci.yml

```
@@ -16,13 +16,13 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x]
+        go-version: [1.16.0]
```

We should make this float with patch releases.

```suggestion
        go-version: [1.16.x]
```